packages/craco/scripts/build.js

```diff
@@ -20,6 +20,17 @@ const context = {
 loadCracoConfigAsync(context).then(cracoConfig => {
     context.paths = getCraPaths(cracoConfig);
 
+    const { isSupported, version } = getReactScriptVersionInformation(cracoConfig);
```

Comment:
I would extract this code in a file, something like `validate-cra-version.js` instead of duplicating it for every script.